packages/wrangler/src/containers/index.ts

```diff
@@ -83,5 +84,12 @@ export const containers = (
 					deleteCommand,
 					containersScope
 				)(args)
+		)
+		.command(
+			"registry",
```

Comment:
```suggestion
			"registries",
```
I'm not very tied to `registries`, I more just wanted to bring it up because it would match `containers` and `images`. It felt a little odd to use `wrangler registry list` and get back multiple registries. But I don't feel very strongly here, just wanted to make sure we think about this first

Reply:
yeah i agree, will change